Tests/PackageGraphPerformanceTests/DependencyResolverPerfTests.swift

```diff
@@ -0,0 +1,54 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright 2016 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import XCTest
+
+import Basic
+import PackageGraph
+
+import struct Utility.Version
+
+import TestSupport
+
+#if ENABLE_PERF_TESTS
+
+class DependencyResolverPerfTests: XCTestCase {
+
+    func testTrivalResolution_X500() throws {
```

Comment:
I'm not sure this actually ends up testing the resolver very much -- it is unlikely to have much setup cost, so I think this is just ending up being a performance test of something lower level. Did you take a look at where it is spending its time? If it isn't in the resolver itself, I would say we should probably change this or write it at a lower level.